torch/_inductor/freezing.py

```diff
@@ -98,9 +98,11 @@ def _freeze(
     # See the details in fx_codegen_and_compile of compile_fx.py.
     view_to_reshape(aot_autograd_gm)
 
-    if tracing_context := torch._guards.TracingContext.try_get():
+    tracing_context = torch._guards.TracingContext.try_get()
+
+    # tracing_context.params_flat_unwrap_subclasses might be None for aot export
+    if tracing_context and tracing_context.params_flat_unwrap_subclasses:
```

Comment:
This is annoying. With the one-pass codegen, I think we can just check aot_inductor on _inductor.config instead of graph.aot_compilation. I will make some refactoring later.